.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+
+> Please describe the issue.
```

Comment:
Maybe that's just me, but I am not sure how useful a file is that'll put "Please describe the issue" into the issue explanation box.

Reply:
Dude I actually laughed out loud when I read your comment, because I was thinking the **exact** same thing to myself as I was writing it. My thought was that we might collaboratively come up with a more useful issue template, though. I'll see if I can find any resources to improve on it, and if not I'll just delete the file.

Reply:
Valid point from @luap42 - but I also approach this as a placeholder file to be improved later.

Reply:
It actually ended up being a non-issue, because it was a legacy method anyways. The suggested method is to create a directory within the new `.github` directory called `ISSUE_TEMPLATES` that you populate with issue templates for bugs, security issues, etc. I just removed the issue template for now while I do some more research on the methodology, but I added links to the documentation in the commit message.